ui/raidboss/data/03-hw/dungeon/the_lost_city_of_amdapor_hard.ts

```diff
@@ -0,0 +1,212 @@
+      netRegex: NetRegexes.startsUsing({ id: '16C7', source: 'Void Monk' }),
+      response: Responses.stunOrInterruptIfPossible(),
+    },
+    {
+      id: 'LostCityHard Void Monk Sucker',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '16C5', source: 'Void Monk', capture: false }),
+      response: Responses.drawIn(),
+    },
+    {
+      id: 'LostCityHard Void Monk Flood',
+      type: 'Ability',
+      // This is an instant cast followup to Sucker.
+      netRegex: NetRegexes.ability({ id: '16C5', source: 'Void Monk', capture: false }),
+      response: Responses.getOut('info'),
+    },
+    {
+      id: 'LostCityHard Winged Lion Wind Resist Down II Gain',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: '41C' }),
+      infoText: (data, matches, output) => {
+        if (data.me === matches.target)
+          return output.text!();
+      },
+      run: (data, matches) => data.windResistDown[matches.target] = true,
+      outputStrings: {
+        text: {
+          en: 'Avoid Aero',
+        },
+      },
+    },
+    {
+      id: 'LostCityHard Winged Lion Wind Resist Down II Lose',
+      type: 'LosesEffect',
+      netRegex: NetRegexes.losesEffect({ effectId: '41C' }),
+      run: (data, matches) => data.windResistDown[matches.target] = false,
+    },
+    {
+      id: 'LostCityHard Winged Lion Earth Resist Down II Gain',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: '41D' }),
+      infoText: (data, matches, output) => {
+        if (data.me === matches.target)
+          return output.text!();
+      },
+      run: (data, matches) => data.earthResistDown[matches.target] = true,
+      outputStrings: {
+        text: {
+          en: 'Avoid Stone',
+        },
+      },
+    },
+    {
+      id: 'LostCityHard Winged Lion Earth Resist Down II Lose',
+      type: 'LosesEffect',
+      netRegex: NetRegexes.losesEffect({ effectId: '41D' }),
+      run: (data, matches) => data.earthResistDown[matches.target] = false,
+    },
+    {
+      id: 'LostCityHard Winged Lion Ancient Stone',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '15D2', source: 'Winged Lion', capture: false }),
+      response: (data, _matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          pop: {
+            en: 'Pop stone orb',
+          },
+          avoid: {
+            en: 'Avoid stone orb',
+          },
+        };
+
+        if (data.earthResistDown[data.me])
+          return { infoText: output.avoid!() };
+        return { alertText: output.pop!() };
+      },
+    },
+    {
+      id: 'LostCityHard Winged Lion Ancient Aero',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '15CE', source: 'Winged Lion', capture: false }),
+      response: (data, _matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
```

Comment:
It's probably okay, but the first time through I was slightly confused by this being a `startsUsing` trigger as opposed to `ability`, since the orb doesn't appear until the castbar completes.